run.sh

```diff
@@ -15,6 +14,7 @@ go run -ldflags "${GOLDFLAGS}" cmd/skycoin/skycoin.go \
     -launch-browser=true \
     -enable-wallet-api=true \
     -rpc-interface=false \
+    -web-interface-port $(($RANDOM%65563))  \
```

Comment:
* Ports below 1000 can only be bound by root
* Port should make sure nothing else is using it
* Should use a CSPRNG-generated number if possible
* Need to do this for electron as well